Header header
uint64 trackingId
JointPositionAndState[] jointPositions
